spec/models/ontology_version_spec.rb

```diff
@@ -34,6 +34,26 @@
       stub_hets_for(hets_out_file('pizza'))
     end
 
+    context 'in subdirectory' do
+      let(:ontology) { create :ontology, basepath: 'subdir/pizza' }
+      let(:qualified_locid) do
+        "http://#{Settings.hostname}#{ontology_version.locid}"
+      end
+
+
+      before do
+        ontology_version
+        allow_any_instance_of(Hets::ParseCaller).to receive(:call) do |iri, *args|
```

Comment:
Unused block argument - `args`. If it's necessary, use `_` or `_args` as an argument name to indicate that it won't be used.<br>Line is too long. [82/80]